t/cobrand/bromley.t

```diff
@@ -846,7 +846,7 @@ subtest "comment on a closed echo report result in a resend under 'Referred to V
     my $req = Open311->test_req_used;
     my $c = CGI::Simple->new($req->content);
     is $c->param('attribute[Event_ID]'), $event_id, 'old event ID included in attributes';
-    like $c->param('description'), qr/Closed report has a new comment: comment on closed event/, 'private comments included in description';
+    like $c->param('description'), qr/comment on closed event|Bromley pkg-tcobrandbromleyt-bromley\@example.com|echo report Test 1 for 1 Detail/, 'Comment on closed report included in new report description';
```

Comment:
Is this meant to have regex alternative `|` like this? It'll match if any of them are present, which doesn't sound right, they should presumably be escaped.